bower.json

```diff
@@ -5,20 +5,18 @@
   "author": "Joe Allan Muharsky <jmuharsky@gmail.com>",
   "dependencies": {},
   "devDependencies": {
-    "angular": "~1.4.7",
-    "angular-mocks": "~1.4.7",
+    "angular": "~1.4.8",
```

Comment:
Generally, it's cleaner to keep version bumps separate from added dependencies - if a new version of a dependency breaks something, rollbacks or version bisections woudl get confusing. I'm not asking you to split them apart in this case, just something to consider for the future.